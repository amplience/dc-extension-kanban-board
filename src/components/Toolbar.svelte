<script>
  import Divider from './Divider.svelte';
</script>

<style lang="scss">
  .toolbar {
    margin: 0 8px 8px 8px;
    height: 40px;
    font-size: 13px;
  }

  .inner {
    height: 100%;
    display: flex;
    align-items: center;
  }

  .label {
    color: #999;
    margin-right: 3px;
  }
</style>

<div class="toolbar">
  <div class="inner">
    <span class="label">Sorted by</span>
    <span>Last modified (most recent first)</span>
  </div>
  <Divider />
</div>
