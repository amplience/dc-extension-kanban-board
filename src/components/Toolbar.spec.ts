import { render } from '@testing-library/svelte';
import Toolbar from './Toolbar.svelte';

describe('toolbar component', () => {
  it('should render a toolbar component', () => {
    const { container } = render(Toolbar, {
      // no-op.
    });

    expect(container.firstChild).toMatchSnapshot();
  });
});
