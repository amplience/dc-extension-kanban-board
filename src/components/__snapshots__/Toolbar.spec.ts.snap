// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`toolbar component should render a toolbar component 1`] = `
<div>
  <div
    class="toolbar svelte-d3be3"
  >
    <div
      class="inner svelte-d3be3"
    >
      <div
        class="sortInfo svelte-d3be3"
      >
        <span
          class="label svelte-d3be3"
        >
          Sorted by
        </span>
         
        <span>
          Last modified (most recent first)
        </span>
      </div>
       
       
      <section
        class="svelte-1ggh271"
      >
        <div
          class="assignee-filter svelte-1ggh271"
        >
          <div
            class="filter-icon  svelte-1ggh271"
            data-testid="filter-icon"
          >
            <div
              class=" svelte-zpbvvv"
              style="--icon-width: 20px; --icon-height: 20px;"
            >
              <object
                src="ic-filter.svg/"
              />
            </div>
          </div>
           
          <span
            class="svelte-1ggh271"
          >
            Assignee
          </span>
           
          <div
            class="assignee-names svelte-1ggh271"
          >
            <div
              class=" clickable  svelte-h28tab"
              data-testid="chip"
            >
              <span
                class="label svelte-h28tab"
              >
                All
              </span>
               
            </div>
          </div>
        </div>
         
      </section>
    </div>
     
    <div
      class="divider horizontal svelte-1rcmn0v"
    />
  </div>
</div>
`;
